scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -113,7 +123,52 @@ workflow GvsExtractCallset {
   }
 }
 
-################################################################################
+task ValidateFilterSetName {
+  input {
+    String filter_set_name
+    String data_project
+    String data_dataset
+    String query_project
+    String? service_account_json_path
+  }
+
+  String has_service_account_file = if (defined(service_account_json_path)) then 'true' else 'false'
+
+  command <<<
+    set -e
+
+    if [ ~{has_service_account_file} = 'true' ]; then
+      gsutil cp ~{service_account_json_path} local.service_account.json
+      gcloud auth activate-service-account --key-file=local.service_account.json
+      gcloud config set project ~{query_project}
+    fi
+
+    echo "project_id = ~{query_project}" > ~/.bigqueryrc
+
+    OUTPUT=$(bq --location=US --project_id=~{query_project} --format=csv query --use_legacy_sql=false "SELECT filter_set_name as available_filter_set_names FROM ~{data_project}.~{data_dataset}.filter_set_info GROUP BY filter_set_name")
+    FILTERSETS=${OUTPUT#"available_filter_set_names"}
+
+    if [[ $FILTERSETS =~ "~{filter_set_name}" ]]; then
+      echo "Filter set name '~{filter_set_name}' found."
+    else
+      echo "ERROR: '~{filter_set_name}' is not an existing filter_set_name. Available in ~{data_project}.~{data_dataset} are"
+      echo $FILTERSETS
+      exit 1
+    fi
+  >>>
+  output {
```

Comment:
I'm hesitant to add more validation than was requested by users into individual WDLs because, I think it makes more sense, once we have one WDL to rule them all, to have an all-in-one validation step that sanity checks all the inputs before running.